model/abstract_model.py

```diff
@@ -19,6 +19,18 @@ def mainloop(self) -> None:
     @abstractmethod
     def get_schedule(self) -> Schedule:
         pass
+    
+    @abstractmethod
+    def get_user_schedule(self) -> Schedule:
+        pass
+    
+    @abstractmethod
+    def change_current_schedule(self):
+        pass
+    
+    @abstractmethod 
+    def get_current_schedule(self) -> Schedule:
```

Comment:
What is the difference supposed to be between `get_current_schedule()` and `get_user_schedule()`

Reply:
ambiguity in the names for sure, `get_current_schedule()` should return the schedule that is currently being used by the model as the schedule. This is what the UI should be using to get the current schedule.
While `get_user_schedule()` should return an actual user schedule based on one day, currently hard coded.

Reply:
Ok. Would it be possible to add to add doc comments?